src/AutoRest.CSharp/Output/Models/Types/ObjectType.cs

```diff
@@ -14,13 +16,14 @@
 using AutoRest.CSharp.Output.Models.Serialization;
 using AutoRest.CSharp.Output.Models.Shared;
 using AutoRest.CSharp.Utilities;
+using Azure.ResourceManager.Core;
 using Microsoft.CodeAnalysis;
 
 namespace AutoRest.CSharp.Output.Models.Types
 {
     internal class ObjectType : TypeProvider
     {
-        private readonly ObjectSchema _objectSchema;
+        protected readonly ObjectSchema _objectSchema;
```

Comment:
Property and ObjectSchema name.

Reply:
Pinged in teams didn't see a reply, what does this comment mean?
Convert it to a property and change the name to the same as the class type?

Reply:
Correct, we shouldn't have protected fields. And properties should use PascalCase naming.